src/encoded/static/components/item-pages/components/ItemFooterRow.js

```diff
@@ -0,0 +1,315 @@
+'use strict';
+
+import React from 'react';
+import _ from 'underscore';
+import { FileEntryBlock } from '../../browse/components/FileEntryBlock';
+import { expFxn, Schemas } from './../../util';
+import { console, object } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
```

Comment:
Codacy found an issue: ['console' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5995039)